### PR TITLE
ATEC-3218: Tools folder separation

### PR DESCRIPTION
**Pull request by adimeny**
Moving the contents of the Tools folder into a different repository, and adding it as a submodule.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,35 @@ This repository contains a CMake template for Archicad Add-On Development.
 
 [![Build](https://github.com/GRAPHISOFT/archicad-addon-cmake/actions/workflows/build.yml/badge.svg)](https://github.com/GRAPHISOFT/archicad-addon-cmake/actions/workflows/build.yml)
 
+## Cloning the repository
```
Comment (kovacsv) [inline README.md:7]: I would put this section under the Prerequisites section.

**Review by kovacsv — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -4,6 +4,35 @@ This repository contains a CMake template for Archicad Add-On Development.
...
+
+https://github.com/GRAPHISOFT/archicad-addon-cmake-tools is included as a submodule.
+
+- Clone the repository using
+  ```
+  git clone --recurse-submodules
+  ```
+  **OR**
+  
+  Run this command after the **first time** you clone the repository:
```
Comment (kovacsv) [inline README.md:17]: I prefer keeping this doc as simple as possible, probably it's enough to include the clone and pull commands.

File: README.md (hunk 1 above)
Comment (adimeny) [reply README.md]: Resolved

File: README.md (hunk 2 above)
Comment (adimeny) [reply README.md]: Resolved

**Review by kovacsv — APPROVED** [review]
Thank you for the change!

**Review by esfarkas — APPROVED** [review]

**Pull request merged**